docker/CMSRucioClient/scripts/legacydn_converter.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+import os
+
+from rucio.client.accountclient import AccountClient
+
+PROXY = os.getenv('X509_USER_PROXY')
+
+client = AccountClient()
+
+def rfc2253dn(legacy_dn: str) -> str:
+    """Converts legacy slash DB to comma separated DN"""
+    if not legacy_dn.startswith('/'):
+        return legacy_dn
+
+    legacy_dn = legacy_dn.replace(',', r'\,')
+    parts = legacy_dn.split('/')[1:]
+
+    return ','.join(parts[::-1])
+
+
+def get_identities_to_add(identities):
+    # get only legacy dns and convert them to new dn
+    dns_to_check = [rfc2253dn(i['identity']) for i in identities if "/" in i['identity']]
```

Comment:
Maybe .startswith here instead of "/" in? I can't say for sure we won't/don't have identities which do have slashes. Probably not allowed, though.